app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    result = command.split(" ")
```

Comment:
no need to use space as arg in split method, it uses space by default

Reply:
also no need to take values by index, u can use split method to unpack splitted string to variables
`arg1, arg2, arg3 = ur_string.split()`